niworkflows/interfaces/images.py

```diff
@@ -14,6 +14,7 @@
 import nibabel as nb
 import nilearn.image as nli
 from textwrap import indent
+import transforms3d
```

Comment:
We'll need to add `'transforms3d'` to the install_requires:

https://github.com/poldracklab/niworkflows/blob/92e63fe87c7e482d99165d3c181dccb025f66d82/niworkflows/__about__.py#L53-L67